lib/sentry.dart

```diff
@@ -161,7 +161,7 @@ class SentryClient {
 
   @visibleForTesting
   String get postUri =>
-      '${dsnUri.scheme}://${dsnUri.host}/api/$projectId/store/';
+      '${dsnUri.scheme}://${dsnUri.host}:${dsnUri.port}/api/$projectId/store/';
```

Comment:
Let's omit the port number if `dsnUri.scheme == 'http' && dsnUri.port == 80 || dsnUri.scheme == 'https' && dsnUri.port == 443`.

Also, please add unit-tests.